src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -37,9 +37,11 @@
         "symfony/http-client-contracts": "^2.5|^3",
         "symfony/process": "^5.4|^6.0",
         "symfony/routing": "^5.4|^6.0",
+        "symfony/serializer": "^6.3",
```

Comment:
Not sure we want these dependency link in the HTTP Kernel.

Reply:
this is optional dependency